packages/features/users/components/AdminPasswordBanner.tsx

```diff
@@ -3,17 +3,19 @@ import Link from "next/link";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { TopBanner } from "@calcom/ui";
+import { trpc } from "@calcom/trpc/react";
 
 function AdminPasswordBanner() {
   const { t } = useLocale();
   const { data } = useSession();
+  const { data: user, isLoading } = trpc.viewer.me.useQuery();
```

Comment:
@sean-brydon should we add twoFactorEnabled in useSession? It would prevent this extra request with a lot of properties that are not required